app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    ("cat_age", "dog_age", "expected"),
+    [
+        (0, 0, [0, 0]),
+        (0, 0, [0, 0]),
```

Comment:
The test case `(0, 0, [0, 0])` is duplicated. Consider removing one of these duplicate entries to avoid redundancy in your test cases.